Source/TagCellLayout.swift

```diff
@@ -97,41 +98,41 @@ public class TagCellLayout: UICollectionViewLayout {
 	
 	//MARK: - Override Methods
 	
-	override public func prepareLayout() {
+	override open func prepare() {
 		resetLayoutState()
 		setupTagCellLayout()
 	}
 	
-	override public func layoutAttributesForItemAtIndexPath(indexPath: NSIndexPath) -> UICollectionViewLayoutAttributes? {
-		if layoutInfoList.count > indexPath.row {
-			return layoutInfoList[indexPath.row].layoutAttribute
+	override open func layoutAttributesForItem(at indexPath: IndexPath) -> UICollectionViewLayoutAttributes? {
+		if layoutInfoList.count > (indexPath as NSIndexPath).row {
+			return layoutInfoList[(indexPath as NSIndexPath).row].layoutAttribute
 		}
```

Comment:
We can avoid the typecast here since `IndexPath` also has those properties,
```
if layoutInfoList.count > indexPath.item {
 			return layoutInfoList[indexPath.item].layoutAttribute
}
```

Reply:
@soolarh Also can you open a new PR against `swift3.2` branch which I have just pushed since most probably I'll point master to swift4.

Reply:
Nevermind @soolarh, I can change the base too :)